public/css/styles.css

```diff
@@ -11,5 +11,9 @@ td {
 }
 
 .current-cell {
-	background-color: pink;
+	background-color: #F8ABC5;
+}
+
+.last-cell {
```

Comment:
Is "last-cell" the final cell in the column? Or is it the previous cell visited? Choose your class names carefully.